packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -194,3 +196,84 @@ class FileAccessLog extends AccessLog {
   }
 }
 
+/**
+ * Represents the properties needed to define backend defaults
+ */
+export interface BackendDefaults {
+  /**
+   * Client policy for backend defaults
+   *
+   * @default none
+   */
+  readonly clientPolicy?: ClientPolicy;
+}
+
+/**
+ * Represents the properties needed to define a Virtual Service backend
+ */
+export interface VirtualServiceBackendOptions {
+  /**
+   * The Virtual Service this backend points to
+   */
+  readonly virtualService: IVirtualService;
+
+  /**
+   * Client policy for the backend
+   *
+   * @default none
+   */
+  readonly clientPolicy?: ClientPolicy;
+}
+
+/**
+ * Properties for a backend
+ */
+export interface BackendConfig {
+  /**
+   * Config for a Virtual Service backend
+   */
+  readonly virtualServiceBackend: CfnVirtualNode.BackendProperty;
+}
+
+
+/**
+ * Contains static factory methods to create backends
+ */
+export abstract class Backend {
+  /**
+   * Construct a Virtual Service backend
+   */
+  public static virtualServiceBackend(props: VirtualServiceBackendOptions): Backend {
```

Comment:
How about calling this method just `virtualService`? The `Backend` part is kind of implied by the class it's on 🙂.

Reply:
I added `backend` because I though `Backend.virtualService` sounded like it implied creation of a new virtual service. On second thought, I agree and I think we can expect a user to infer this. So I'm fine renaming it.